src/jmcomic/jm_plugin.py

```
@@ -799,6 +799,91 @@ def ensure_make_pdf_dir(pdf_dir: str):
+        self.execute_deletion(img_path_ls)
+
+    def write_img_2_long_img(self, long_img_path, album: JmAlbumDetail, photo: JmPhotoDetail) -> list[str]:
+        import itertools
+        from PIL import Image
+
+        if album is None:
+            img_dir_items = [self.option.decide_image_save_dir(photo)]
+        else:
+            img_dir_items = [self.option.decide_image_save_dir(photo) for photo in album]
+
+        img_paths = itertools.chain(*map(files_of_dir, img_dir_items))
```

Ruff

`files_of_dir` may be undefined, or defined from star imports

(F405)